### PR TITLE
added a blog

### PR DESCRIPTION
**Pull request by csjuXYZ**
Fixes # .

Make sure these boxes are checked before creating your pull request -- thank you!

- [x] Add Blog Header, including title, author, date.
e.g.

``` md
---
 title: "'Score Your Face Photo' a ML&Wechaty practice"
 date: 2017-09-18 09:00 +0800
 author: huyingxi
 ---
 > Author: [@huyingxi](https://github.com/huyingxi/wechaty_selfie) enjoying ML&Wechaty at BUPT
```

- [x] Keep all filenames & url as lowercase, and use `-` to connect words instead of space. e.g. `2017-10-06-wechat-pc-impactor` instead of `2017-10-06-WeChat PC Impactor`,
- [x] Embed the photo & video before publishing, save all external file to the blog `/download/2018` directory.
- [x] Add `<!--more-->` section to just show the abstract in blog homepage.
- [x] Add teaser image for the blog
- [x] Add your self as an author to the author data file at <https://github.com/wechaty/wechaty.github.io/blob/master/_data/authors.yml> with your GitHub username. And please also remember to reference yourself with the Github username in your blog post, so that we can have nice author information besides your blog after we published it!

[See more](https://github.com/wechaty/wechaty.github.io#contribute-guideline)

### CODE REVIEW

**Review by csju — APPROVED** [review]
ok

**Review by csju — APPROVED** [review]
ok

**Review by csju — APPROVED** [review]

File: _posts/2020-07-23-wechaty-lottery.md [hunk 1]
```diff
@@ -0,0 +1,142 @@
+---
+title: "微信群内抽奖机器人(wechaty_lottery)"
+author: csju
+date: "2020-07-23 10:24:00 +0800"
+categories: project
+tags:
+  - wechaty
+  - wechaty-puppet-padplus
+header:
+  teaser: ..\assets\2020\wechaty-lottery\lottery-header.jpg
```
Comment (lijiarui) [inline _posts/2020-07-23-wechaty-lottery.md]: Please remove  `..`

File: _posts/2020-07-23-wechaty-lottery.md [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+### 功能
+
+通过关键词触发机器人随机抽取群内一人获奖,已除去发奖人与机器人
+
+### 逻辑
+
+- 通过"抽奖+空格"格式触发抽奖机器人,例如"抽奖 抱枕","抽奖 一等奖"等  
+- 机器人被触发后获取当前群内成员数组,并去除机器人和发奖人
+- 等概率随机抽取一人,在群内消息通知中奖信息并同时@发奖人和中奖人
+- ![测试](..\assets\2020\wechaty-lottery\pic01.jpg)
```
Comment (lijiarui) [inline _posts/2020-07-23-wechaty-lottery.md]: Please remove `..`

**Review by lijiarui — CHANGES_REQUESTED** [review]

File: _posts/2020-07-23-wechaty-lottery.md (hunk 2 above)
Comment (csjuXYZ) [reply _posts/2020-07-23-wechaty-lottery.md]: ok

**Review by lijiarui — APPROVED** [review]

**Comment by lijiarui** [comment]
Please resolve the conflict, then I could able merge this PR.

**Comment by lijiarui** [comment]
Congratulations! 🎉 🎉 🎉

You become a wechaty contributor successfully! 👏 👏 👏

Please:

- [ ]  Add my wechat: ruirui_0914 and send me this pr link. I will invite you into Wechaty Contributor Room
- [ ] Update your profile here: [Contributor List](https://docs.google.com/spreadsheets/d/1XcDoIczyIclqXP1p90Sz7S0n4Q22xjFzJmjPFlU2g1E/edit#gid=0), then other contributors will know you better!
- [ ]  Join wechaty org by clicking https://github.com/wechaty

**Pull request merged**